common/test/db_cache/bok_choy_schema.sql

```diff
@@ -1810,45 +1810,6 @@ CREATE TABLE `external_auth_externalauthmap` (
   CONSTRAINT `external_auth_externala_user_id_644e7779f2d52b9a_fk_auth_user_id` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
-DROP TABLE IF EXISTS `foldit_puzzlecomplete`;
```

Comment:
Honestly I'd defer to @symbolist or @macdiesel on that one. They just created that cache and would know better about how migrations work nowadays.